src/components/PostDetails.tsx

```diff
@@ -1,117 +1,105 @@
+                Something went wrong
+              </div>
+            )}
 
-          <article className="message is-small" data-cy="Comment">
-            <div className="message-header">
-              <a
-                href="mailto:misha@mate.academy"
-                data-cy="CommentAuthor"
-              >
-                Misha Hrynko
-              </a>
+            {postComments.length > 0 && (
+              <>
+                <p className="title is-4">Comments:</p>
+                {postComments.map((comment: Comment) => {
+                  return (
+                    <SingleComment
+                      comment={comment}
+                      comments={comments}
+                      setComments={setComments}
+                      postComments={postComments}
+                      setPostComments={setPostComments}
+                      setIsError={setIsError}
+                    />
+                  );
+                })}
+              </>
+            )}
 
+            {postComments.length === 0 && (
+              <p className="title is-4" data-cy="NoCommentsMessage">
+                No comments yet
+              </p>
+            )}
+            {!isSeen
+            && (
               <button
-                data-cy="CommentDelete"
+                data-cy="WriteCommentButton"
                 type="button"
-                className="delete is-small"
-                aria-label="delete"
+                className="button is-link"
+                onClick={() => {
+                  setIsSeen(true);
+                }}
               >
-                delete button
+                Write a comment
               </button>
-            </div>
-            <div
-              className="message-body"
-              data-cy="CommentBody"
-            >
-              One more comment
-            </div>
-          </article>
-
-          <article className="message is-small" data-cy="Comment">
-            <div className="message-header">
-              <a
-                href="mailto:misha@mate.academy"
-                data-cy="CommentAuthor"
-              >
-                Misha Hrynko
-              </a>
-
-              <button
-                data-cy="CommentDelete"
-                type="button"
-                className="delete is-small"
-                aria-label="delete"
-              >
-                delete button
-              </button>
-            </div>
-
-            <div className="message-body" data-cy="CommentBody">
-              {'Multi\nline\ncomment'}
-            </div>
-          </article>
-
-          <button
-            data-cy="WriteCommentButton"
-            type="button"
-            className="button is-link"
-          >
-            Write a comment
-          </button>
-        </div>
-
-        <NewCommentForm />
+            )}
+          </div>
+        )}
+        {
+          isSeen && (
```

Comment:
Check it everywhere , please 
```suggestion
        {isSeen && (
```